doc.js

```diff
@@ -225,41 +245,50 @@ define('doc', ['event'], function(event) {
 			},
 
 			'addClass' : function(clazz) {
+				var clazzList = clazz.split(' ');
 				this.each(function(el) {
-					if(!el.classList) {
-						if(!fallbackHasClass(el, clazz)) {
-							el.className += ' ' + clazz + ' ';
+					clazzList.forEach(function(clazz) {
+						if(!el.classList) {
+							if(!fallbackHasClass(el, clazz)) {
+								el.className += ' ' + clazz + ' ';
+							}
+						} else {
+							el.classList.add(clazz);
 						}
-					} else {
-						el.classList.add(clazz);
-					}
+					});
 				});
 				return this;
 			},
 
 			'removeClass' : function(clazz) {
+				var clazzList = clazz.split(' ');
 				this.each(function(el) {
-					if(!el.classList) {
-						el.className = el.className.replace(new RegExp('(^|\\s)' + clazz + '($|\\s)'), '');
-					} else {
-						el.classList.remove(clazz);
-					}
+					clazzList.forEach(function(clazz) {
+						if(!el.classList) {
+							el.className = el.className.replace(new RegExp('(^|\\s)' + clazz + '($|\\s)'), '');
+						} else {
+							el.classList.remove(clazz);
```

Comment:
`classList.remove` also seems to support multiple classes

Reply:
![image](https://user-images.githubusercontent.com/1116568/30566521-c0e3efca-9ca2-11e7-995f-aee6d3b4009e.png)

Reply:
😢